package.json

```diff
@@ -31,37 +31,37 @@
     "boom": "4.2.0",
     "chalk": "^1.1.3",
     "coveralls": "2.11.15",
-    "electrode-csrf-jwt": "1.1.2",
+    "electrode-csrf-jwt": "1.2.0",
     "escape-html": "1.0.3",
-    "eslint": "3.12.2",
-    "eslint-plugin-react": "6.8.0",
+    "eslint": "3.13.1",
+    "eslint-plugin-react": "6.9.0",
     "good": "7.1.0",
     "good-console": "6.2.0",
     "good-squeeze": "5.0.1",
     "hapi": "16.1.0",
-    "history": "2.1.2",
+    "history": "4.5.1",
```

Comment:
so if you manually update the package.json on this branch and change it back to 2.1.2, it seems like greenkeeper refrains from updating major package versions on incremental PRs. I think on the first one it just tries to take the latest of everything.